### ISSUE TITLE
UnicodeEncodeError: 'charmap' codec can't encode character '\u2502' in position 559: character maps to <undefined>

### ISSUE THREAD

**Issue by ccornici** [pypa/pipx#843]
**Describe the bug**

See title

**How to reproduce**

```
CLC@DESKTOP-4CR54IP MINGW64 /c/Users/CLX
$ pipx install eth-brownie

Traceback (most recent call last):
  File "C:\Users\CLX\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\CLX\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\Scripts\pipx.exe\__main__.py", line 7, in <module>
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 779, in cli
    return run_pipx_command(parsed_pipx_args)
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 202, in run_pipx_command
    return commands.install(
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\site-packages\pipx\commands\install.py", line 60, in install
    venv.install_package(
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\site-packages\pipx\venv.py", line 238, in install_package
    subprocess_post_check_handle_pip_error(pip_process)
  File "C:\Users\CLX\AppData\Roaming\Python\Python310\site-packages\pipx\util.py", line 349, in subprocess_post_check_handle_pip_error
    print(completed_process.stderr, file=pip_error_fh, end="")
  File "C:\Users\CLX\AppData\Local\Programs\Python\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2502' in position 559: character maps to <undefined>
```

I tried the env vars from here and didn't help

https://stackoverflow.com/questions/27092833/unicodeencodeerror-charmap-codec-cant-encode-characters

This is on Win 10.

```
$ python --version
Python 3.10.4

$ pipx --version
1.0.0
```

**Comment by meowmeowmeowcat** [comment]
Duplicate of https://github.com/pypa/pipx/issues/819

**Comment by turazashvili** [comment]
FOUND SOLUTION!
Deleted python 3.10.4 >>> Restarted system >>> Installed v3.9.10 >>> Run again install pip, pipx ensurepath (it gave me path error, but i just inserted the shortcut for python.exe wherever it was looking for it  >>> finally run pipx install eth-brownie and it Worked!
<img width="264" alt="image" src="https://user-images.githubusercontent.com/74835523/168460085-5de8d905-9c0e-4aaa-a191-de85d740969a.png">


Hoping it helps while they rolling the fix for latest Python version.

**Comment by cs01** [comment]
This has been fixed in v1.1.0

**Comment by turazashvili** [comment]
> This has been fixed in v1.1.0

Thanks for update.

it's well known thing, but I feel like it's more challenging to set up an environment than creating a project :))))
